warehouse/packaging/models.py

```diff
@@ -92,6 +93,10 @@ class Project(db.ModelBase):
         nullable=False,
         server_default=sql.func.now(),
     )
+    updated = Column(
+        DateTime(timezone=False),
+        onupdate=datetime.datetime.now
```

Comment:
We're going to deploy Warehouse and PyPI side by side for awhile, so keeping as much as possible as far as data integrity and update functionality will help ensure a consistent behavior during that time that both things are accessing the database.

Beyond that time period, I just have a preference for using the abilities of the database where possible, I'm not really concerned about performance penalty, just that I feel it's cleaner since those actions will work even when you're not going through the ORM so it's less likely that they won't happen.